functorch/test/test_ops.py

```diff
@@ -1128,37 +1120,8 @@ def reference(primals, cotangents, primals_tangents, cotangents_tangents):
                     expected = (tree_unflatten(primals_out, spec), tree_unflatten(tangents_out, spec))
                 return expected
 
-            # HACK: obviously pytorch should also have the same coverage
-            # For things that do have the same coverage, we test that jvp x vjp
-            # are the same between PyTorch and functorch. For things that don't,
-            # we check that jacfwd(vjp) and jacrev(vjp) are the same. This results
-            # in slower tests.
-            FUNCTORCH_HAS_FORMULA_BUT_NOT_PYTORCH = {
-                'nn.functional.nll_loss',
-                'softmax',
-                'log_softmax',
-                'nn.functional.cross_entropy',
-                'nn.functional.layer_norm',
-                'nn.functional.batch_norm',
```

Comment:
(1) 🥳 this is awesome
(2) I think I'm seeing that we're still xfailing `nll_loss`, `cross_entropy` and `batch_norm` (but not instance norm, which is just reshapes and calls batch norm 🤔 ). Just wanted to check that this matches what your seeing before I file an issue to investigate this (doesn't have to be done in this PR)